docs/api/http.md

```diff
@@ -216,7 +216,7 @@ import { serve, file } from "bun";
 
 serve({
   fetch(req) {
-    return new Response(Bun.file("./hello.txt"));
+    return new Response(file("./hello.txt"));
```

Comment:
I find it clearer to consistently keep `Bun.serve` and `Bun.file` in the documentation

Reply:
@colinhacks what do you think? I agree with that we should make all the examples on here use the full `Bun.serve`/`Bun.file`